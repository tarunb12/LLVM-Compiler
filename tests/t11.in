// Recursion test (fibonacci)

int fibonacci(int c) {
    if (c <= 1) return c
    else return fibonacci(c - 1) + fibonacci(c - 2)
}

int main() {
    int i
    for (i = 0; i < 13; i++) {
        printf("fibonacci(%d) = %d\n", i, fibonacci(i))
    }
    return 0
}