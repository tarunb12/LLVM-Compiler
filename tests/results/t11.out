gcd(30, 45) = 15
gcd(8, 23) = 1
gcd(36, 404) = 4
gcd(83, 93) = 1
gcd(203, -1102) = 29
gcd(163231, 135749) = 151
